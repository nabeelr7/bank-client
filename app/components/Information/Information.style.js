import styled from 'styled-components';
import { media, colors } from 'utils';

const StyledInformation = styled.div`
  max-width: 1260px;
  margin: 10px auto 0;
  color: ${colors.white};
  background-color: rgb(0, 152, 219);
  padding: 10px 15px;
  border-radius: 2px;

  ${media.tablet`
    margin: 10px auto;
    padding: 10px 130px;
  `}
`;

export default StyledInformation;
