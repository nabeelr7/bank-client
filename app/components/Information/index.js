/**
 *
 * Information
 *
 */

import React from 'react';

// import { FormattedMessage } from 'react-intl';
// import messages from './messages';
import StyledInformation from './Information.style';

export default function Information() {
  return (
    <StyledInformation>
      If you take advantage of our promotion and register your account by the
      end of September in our bank, you will receive 10,00 USD from us. Accounts
      already created will also receive this from us.
    </StyledInformation>
  );
}
