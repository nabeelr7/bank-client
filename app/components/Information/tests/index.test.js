import React from 'react';
import { render } from '@testing-library/react';

import { createMemoryHistory } from 'history';
import { Provider } from 'react-redux';
import { IntlProvider } from 'react-intl';
import Information from '../index';
import { DEFAULT_LOCALE } from '../../../locales';
import configureStore from '../../../configureStore';

describe('<Information />', () => {
  const history = createMemoryHistory();
  const store = configureStore({}, history);

  it('should render a information', () => {
    const { container } = render(
      <Provider store={store}>
        <IntlProvider locale={DEFAULT_LOCALE}>
          <Information />
        </IntlProvider>
      </Provider>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
