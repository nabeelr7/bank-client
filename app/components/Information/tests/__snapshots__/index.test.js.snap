// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Information /> should render a information 1`] = `
<div
  class="Informationstyle__StyledInformation-sc-7iwlvw-0 eEPUpK"
>
  If you take advantage of our promotion and register your account by the end of September in our bank, you will receive 10,00 USD from us. Accounts already created will also receive this from us.
</div>
`;
